packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -234,6 +234,12 @@ export type CallCompositeOptions = {
      */
     layout?: VideoGalleryLayout;
   };
+  /* @conditional-compile-remove(end-of-call-survey) */
+  /**
+   * Show call survey at end of call
+   * @defaultValue true
```

Comment:
we should name it to default to false, so instead of survey, have hideSurvey. (see other comment about adding more survey options also, maybe `hideSurvey`/`hidden`/`doNotShow` is inside a surveyOptions bag)

Reply:
I was following what was done with another feature errorBar, where we have it default true and when passing into false, error bars will not show. Survey is similar to errorBar since they are both optional features, so I followed the naming convention